README.md

```diff
@@ -48,6 +54,105 @@ bot.readyCommand({
     code: `$log[Ready on $userTag[$clientID]]`
 })
 ```
+## Additional Settings
+
+### Variables
+
+- What are variables?
+
+Variables are Key-Value based data which is stored in the database, useful for Economy and Leveling system as it is allows you to save data.
+
+```js
+bot.variables({
+  VariableName1: "Value", //Returns "Value"
+  VariableName2: "Value2" //Returns "Value2"
+})
+```
+
+### Callbacks
+
+- What are callbacks?
+
+It's simple and easy process, it essentially allows you to trigger events, such as user joining a Guild. This will trigger an event, causing commands with supported type for each callbacks to be executed such as.
+
+```js
+bot.joinCommand({ //command
+channel: "channel id", //channel where it will log
+code: `your code` //Message sent to <channel>
+})
+bot.onJoin()
+```
+
+### Slash Commands
+
+- With easy and simple functions, you can make Slash Commands with your Bots quick!
+
+```js
+bot.command({
+name: "create",
+code: `$createApplicationCommand[$guildID;aoijs;a cool slash command for aoi.js;true]`
+/*
+    Code Breakdown:
+This will make a slashcommand named "aoijs" (meaning you'd do /aoijs),
+the description will say "a cool slash command for aoijs"
+*/
+})
+
+// Initializing the slash command
+
+bot.interactionCommand({
+name: "aoijs", 
+prototype : 'slash',
+code: `$interactionReply[AOIjs is an awesome package!]`
+/*
+The code will be execute once /aoijs has been ran
+*/
+})
+bot.onInteractionCreate()
+
+```
+
+### Custom Functions for Intermediate Developers
+- With the latest feature of aoi.js v5.0.0 and above, which enables Developers to create their own custom function built-in and easy.
+
+```js
+/*THIS IS JUST AN EXAMPLE IN YOUR MAIN FILE*/
+
+const aoijs = require("aoi.js")
+
+const bot = new aoijs.Bot({
+token: "DISCRD BOT TOKEN",
+prefix: "DISCRD BOT PREFIX",
+intents: ["GUILDS", "GUILD_MESSAGES"]
+})
+
+//Events
+bot.onMessage()
+
+/*CREATING THE ACTUAL FUNCTION*/
+/*EXAMPLE OF MAKING $authorOnlyButton*/
+
+bot.functionManager.createCustomFunction({
+name : '$authorOnlyButton', //FUNCTION NAME 
+params : ['index','label','style','customId','disabled','emoji'],//THE TYPE OF PARAMS
+type : 'aoi.js', //TYPE METHOD
+code : ` 
+$addButton[{index};{label};{style};{customId}_$authorId;{disabled};{emoji}]
+` //THE ACTUAL CODE IT WILL BE RETURN
+})
+
+/*ONLY EXPERIENCED WITH UNDERSTANDING OF AOIJS SHOULD USE*/
+
+/*BY USING CUSTOM FUNCTION WE ARE'T OBLIGED OF WHAT HAPPENS TO YOUR CLIENT*/
+
+/*WITH THIS FUNCTION MANAGER IT JUST CREATED $authorOnlyButton function*/
+```
+
+
+## Music
+
+With our new addition of music properties to Aoi.js, you can use music features in v5 too.
+Check out the [docs](https://akaruidevelopment.github.io/music/) for more information!
```

Comment:
This redirect link should be the actual documentation (aoi.js) (once the PR is merged)